docs/data/base/components/button/button.md

```diff
@@ -39,7 +39,7 @@ Notice that the second button cannot be clicked due to the `disabled` prop:
 The Button component is composed of a root `<button>` slot with no interior slots:
 
 ```html
-<button class="MuiButton-root">
+<button class="base-Button-root">
```

Comment:
It's a personal preference, but I would rather write `BaseButton-root`, than `base-Button-root`.

Reply:
What's the plan for Material UI class names, `.mui-Button-root`? If we don't plan to change it,  
`.BaseButton-root` looks like a clear direction to take. It's arbitrary, so we might as well be consistent.